packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx

```diff
@@ -73,22 +85,21 @@ export const MetricPeekModal: FC<Props> = ({ opened, onClose }) => {
             case MetricExplorerComparison.DIFFERENT_METRIC:
                 return {
                     type: MetricExplorerComparison.DIFFERENT_METRIC,
-                    metricName: metricQuery.data.name,
+                    // TODO: this is hardcoded for now, should be a dropdown in the UI
+                    metricTable: 'orders',
+                    metricName: 'total_non_completed_order_amount',
```

Comment:
upcoming work for part 2 (ui)

Reply:
and I'm assuming that the legends + tooltips will be addressed in part 2, right?

Reply:
yes